x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -128,23 +123,6 @@ function AlertsPage() {
 
   const manageRulesHref = http.basePath.prepend('/app/observability/alerts/rules');
 
-  const dynamicIndexPatternsAsyncState = useAsync(async (): Promise<DataViewBase[]> => {
-    if (indexNames.length === 0) {
-      return [];
-    }
-
-    return [
-      {
-        id: 'dynamic-observability-alerts-table-index-pattern',
-        title: indexNames.join(','),
-        fields: await dataViews.getFieldsForWildcard({
-          pattern: indexNames.join(','),
-          allowNoIndex: true,
-        }),
-      },
-    ];
```

Comment:
is this now abstracted in the `dataViews.create` service?

Reply:
Yes, exactly.
Previously, we didn't have `dataViews.create` so it was implemented this way in the meantime but now we have `dataViews` service that can be used instead of manually creating it.